storops/unity/resource/lun.py

```diff
@@ -289,6 +326,55 @@ def snapshots(self):
                              storage_resource=self.storage_resource)
 
 
+class _CGLunProxy(object):
+    """Proxy any modify request of member LUN to the UnityConsistencyGroup"""
```

Comment:
Why do we need a proxy class to handle member LUN modification requests? Why not just modify it in UnityLun itself?

Reply:
It's the limitation of rest API: 
any modification of LUN in cg needs to use `ModifyConsistencyGroup` of `storageResource`, what's more, the parameters differs a lot than a standalone LUN.